### PR TITLE
style: adjust hcp icon tile logo color

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ksupdate-hcp-tile-hashicorp.vercel.app/hcp/docs) 🔎
- [Asana task](https://app.asana.com/0/1202702246158644/1202946807363621) 🎟️
- [Figma](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=12456%3A146008) 🎨 

## 🗒️ What

This PR updates the hcp icon tile logo styles to match the new spec. 

## 🧪 Testing

- visit the [hcp docs landing](https://dev-portal-git-ksupdate-hcp-tile-hashicorp.vercel.app/hcp/docs), verify the tile logo styles matches the spec

### CODE REVIEW

File: src/components/icon-tile/icon-tile.module.css [hunk 1]
```diff
@@ -59,87 +59,91 @@ but is necessary for us to implement the current designs for the
...
-  --icon-color: var(--token-color-foreground-high-contrast);
-  --border-color: var(--token-color-foreground-strong);
-  --background: var(--token-color-foreground-primary);
+	--icon-color: var(--token-color-foreground-high-contrast);
+	--border-color: var(--token-color-foreground-strong);
+	--background: linear-gradient(
+		315deg,
+		var(--token-color-palette-neutral-700) 0%,
+		var(--token-color-palette-neutral-600) 100%
+	);
```
Comment (kendallstrautman) [inline src/components/icon-tile/icon-tile.module.css:68]: This is the meaning change to this file, beyond formatting.

Comment (BRKalow) [reply src/components/icon-tile/icon-tile.module.css:68]: https://github.com/hashicorp/dev-portal/pull/975/files?w=1

Comment (kendallstrautman) [reply src/components/icon-tile/icon-tile.module.css:68]: ahh the whitespace trick! ty I forgot the shorthand

**Review by BRKalow — APPROVED** [review]
works 4 me

**Pull request merged**